src/Chapter1.hs

```diff
@@ -635,8 +653,8 @@ Implement a function that returns the first digit of a given number.
 You need to use recursion in this task. Feel free to return to it later, if you
 aren't ready for this boss yet!
 -}
-
-firstDigit n = error "firstDigit: Not implemented!"
+firstDigit :: Int -> Int
+firstDigit n = if div (abs n) 10 == 0 then abs n else firstDigit (div (abs n) 10)
```

Comment:
Great one! 💯 This is nice and accurate code!

One thing to notice is that you use `abs n` several times. Usually, when this is the case, it means that you can put it into the helper variable in `let`, for example, and reuse it in all three places.